web/src/common/styles/authentik.css

```diff
@@ -1,5 +1,9 @@
 /* #region Global */
 
+.__AK_BASE_STYLES__ {
+    --__AK_BASE_STYLES__: 1;
+}
+
```

Comment:
Added to better debug constructed stylesheets via [`inspectStyleSheetTree`](https://github.com/goauthentik/authentik/blob/ae2af6e58e4f7f4255f804c18188b50f80c5018b/web/src/common/stylesheets.ts#L176)

<img width="583" alt="Screenshot 2025-05-09 at 02 15 29" src="https://github.com/user-attachments/assets/27473573-7013-4842-b473-af51a86cc4b6" />

Reply:
Should probably add a comment to the code why that's there